TypeCobol.Test/GrammarTest.cs

```diff
@@ -87,8 +88,9 @@ private static void AppendTextToFiles(string tetxToAppend, params string[] files
 			    {
 			        ExecToStep = ExecutionStep.CrossCheck,
 #if EUROINFO_RULES
-			        AutoRemarksEnable = autoRemarks
+			        AutoRemarksEnable = autoRemarks,
 #endif
+			        CheckEndAlignment = TypeCobolCheckOption.Parse("ignore")       // no check end alignment for third party tests
```

Comment:
Why ignore here ?

Reply:
In GrammarTest, argument ignoreWarningDiag is set to true; 
this line had the same action than code, more far : 
if (ignoreWarningDiag)   diagnostics = diagnostics.Where(d => d.Info.Severity != Severity.Warning).ToList();
This line has been deleted
https://github.com/TypeCobolTeam/TypeCobol/pull/1485/commits/887743a9e47d95e16fa3eda225d6f1f01ad78248